README.md

```diff
@@ -1,2 +1,114 @@
-# pymockserver
-MockServer implementation in Python
+# PyMockserver
+
+[![Release](https://github.com/kudlatyamroth/pymockserver/actions/workflows/release.yml/badge.svg)](https://github.com/kudlatyamroth/pymockserver/actions/workflows/release.yml)
+[![Unittests](https://github.com/kudlatyamroth/pymockserver/actions/workflows/test.yaml/badge.svg)](https://github.com/kudlatyamroth/pymockserver/actions/workflows/test.yaml)
+
+# Introduction
+
+One more implementation of MockServer, but for simple tasks.
+It aims to be simple and fast.
+
+Main differences to other solutions like `https://www.mock-server.com/`:
+- easier to create mock
+- easier to delete mock
+- faster create and get mock for request
+
+# Usage
+
+### Create mock
+
+To create mock, just send post request to PyMockserver:
+
+```shell
+curl --request POST "http://pymockserver/mockserver" --data '{
+        "httpRequest": {
+            "method": "GET",
+            "path": "/test"
+        },
+        "httpResponse": {
+            "body": {
+                "status": "ok"
+            },
+            "statusCode": 200,
+            "remainingTimes": -1,
+            "delay": 0
+        }
+    }'
+```
+
+And now to request for this mock like that:
+```shell
+curl --request GET "http://pymockserver/test"
+```
+and this will return status code 200 and body:
+```json
+{
+    "status": "ok"
+}
+```
+
+### Delete mock
+
+Same as create mock, but request method DELETE. So to delete previous mock send request:
+```shell
+curl --request DELETE "http://pymockserver/mockserver" --data '{
+        "method": "GET",
+        "path": "/test"
+    }'
+```
+
+### Delete all mock
+
+To delete all mocks, just send delete request to `/mockserver/reset`:
+```shell
+curl --request DELETE "http://pymockserver/mockserver/reset"
+```
+
+
+### Get all mock
+
+To get all mocks, just send get request to `/mockserver`:
+```shell
+curl --request GET "http://pymockserver/mockserver"
+```
+
+### Fixtures
+
+PyMockserver has ability to load fixtures.
+To provide fixtures for PyMockserver, just set fixtureFiles in helm values file, ex:
+```yaml
+fixtureFiles:
+  fixtures.yaml: |
+    - httpRequest:
+        method: GET
+        path: /test
+        queryStringParameters:
+      httpResponse:
+        statusCode: 200
+        body:
+          status: "ok"
+```
+
+## Request options
+
+httpRequest:
+
+| key    | example value | default | required | description                                                          |
+|--------|---------------|---------|----------|----------------------------------------------------------------------|
+| method | `"POST"`        | GET | - | Specify valid http request method, with whitch mock will be returned |
```

Comment:
whitch -> which

sorry for being picky :)

Reply:
thanks for being picky :)